tests/test_superruns.py

```diff
@@ -0,0 +1,247 @@
+import os
+import unittest
+import shutil
+import strax
+import numpy as np
+import tempfile
+from strax.testutils import Records, PeaksWoByRunDefault
+import datetime
+import pytz
+import json
+from bson import json_util
+import re
+
+
+class TestSuperRuns(unittest.TestCase):
+
+    def setUp(self, superrun_name='_superrun_test'):
+        self.offset_between_subruns = 10
+        self.superrun_name = superrun_name
+        self.tempdir = tempfile.mkdtemp()
+        self.context = strax.Context(storage=[strax.DataDirectory(self.tempdir,
+                                                                  provide_run_metadata=True,
+                                                                  readonly=False,
+                                                                  deep_scan=True)],
+                                     register=[Records, RecordsExtension],
+                                     use_per_run_defaults=False)
+        self.context.set_context_config({'write_superruns': True})
+        
+        logger = self.context.log
+        logger.addFilter(lambda s: not re.match(".*Could not estimate run start and end time.*",
+                                                s.getMessage()))
+
+        self._create_subruns()
+        self.context.define_run(self.superrun_name, data=self.subrun_ids)  # Define superrun
```

Comment:
`data` should be a DataFrame or an array right, now it appears to be a list instead. After seeing this I added this other comment about the docs of `define_run`